src/util/interval_template.h

```diff
@@ -143,6 +143,47 @@ template<class T> class interval_templatet
     }
   }
 
+  void make_bottom()
```

Comment:
:question: I need to see why these changes are introduced - they seem unrelated to the rest of the PR>